apps/style/code-studio/levelbuilder.scss

```diff
@@ -178,3 +178,13 @@ table.checkboxes {
     background-position: -313px -119px;
   }
 }
+
+.alignment-box{
```

Comment:
does `box` in this content refer to something specific? If not, could this perhaps be named something that uses existing terms like `aligned-input-group`?